dashboard/app/models/user.rb

```diff
@@ -76,7 +76,7 @@
 
 class User < ActiveRecord::Base
   include SerializedProperties
-  serialized_attrs %w(ops_first_name ops_last_name district_id ops_school ops_gender)
+  serialized_attrs %w(ops_first_name ops_last_name district_id ops_school ops_gender races race_interstitial_shown)
```

Comment:
Yup - setting that up right now, blocking the user from opting out in addition to selecting other items.